services/localoam.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var meta = require('../controllers/meta.js');
+var request = require('request');
+
+/**
+* LocalOAM Constructor that handles intractions with local OAM bucket server
+*
+* @constructor
+* @param {String} bucket (optional) - Local OAM Bucket URL. Can be set by LOCAL_OAM_BUCKET env var.
+*/
+var LocalOAM = function (bucket) {
+  this.params = {
+    bucket: bucket || process.env.LOCAL_OAM_BUCKET
+  };
+};
+
+/**
+* Read bucket method for LocalOAM. It reads the LocalOAM bucket and adds all the `.json` metadata to Meta model
+*
+* @param {responseCallback} cb - The callback that handles the response
+* @param {finishedCallback} finished - The callback that handles when reading is done
+*/
+
+LocalOAM.prototype.readBucket = function (lastSystemUpdate, cb, done) {
```

Comment:
`cb` and `done` sound redundant, so it's unclear what each is for (other than the order of arguments). Perhaps `cb` should be `each` or something to indicate that it should be called multiple times, once for each entry?

Also, the parameter names in the jsdoc don't match.